src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -46,4 +53,15 @@ public OrderListGetDto acceptOrders(@RequestBody @Valid OrderListPostDto orderLi
                                         @PathVariable(value = "franchiseeId") Long franchiseeId) {
         return franchiseeService.acceptOrders(orderListPostDto);
     }
+
+    @GetMapping("/{franchiseeId}/orders")
+    @ResponseStatus(HttpStatus.OK)
+    public List<OpenOrderGetDto> getOrders(@PathVariable Long franchiseeId, @RequestParam @NotNull String status){
+        if (OrderStatus.ASSIGNED_PENDING.getStatus().equals(status)) {
+            return franchiseeService.getFirst10OpenOrders(franchiseeId);
+        } else {
+            throw new OrderStatusInvalidException("order status invalid");
```

Comment:
I pulled your code and ran the test, this exception seems to be not covered

Reply:
Yes, my bad.
I'll add it to test.
Thanks.